aggregate/housing_security/rent_median.py

```diff
@@ -56,14 +56,14 @@ def rent_median(geography: str, write_to_internal_review=False) -> pd.DataFrame:
     final_hh = final_hh.reindex(
         columns=order_PUMS_QOL_multiple_years(
             categories=["units_payingrent"],
-            measures=["_count", "_count_moe", "_count_cv"],
+            measures=["_count", "_count_moe", "_count_cv", "_pct", "_pct_moe"],
             years=["_0812", "_1519"],
         )
     )
 
     final = pd.concat([final_md, final_hh], axis=1)
 
-    final.dropna(axis=1, how="all", inplace=True)
+    final["units_payingrent_1519_pct"] = 100
```

Comment:
Ok looked over how we are setting this column and my understanding is that we have this `EDDT_UnitsAffordablebyAMI_2015-2019.xlsx` file that has this `ROcc2` percent field populated while the ACS PUMS spreadsheets don't? And this pct columns populated with 100's is more correct than the `ROcc2` or `HUPRt` columns in the ACS data?

Reply:
Hey @SashaWeinstein  didn't meant it to be more confusing but I made a new change and if you want to hear the full explanation for impute the value and keep the column I am happy to talk over a call but it is a little more convoluted

Reply:
Ok if you and Amanda feel good about it I don't want it to hold us up, feel ok approving without understanding exactly what's going on